src/components/ads/ad-utils/pbhAdFunctions.js

```diff
@@ -0,0 +1,70 @@
+import { AD_TYPES, AD_FUNCS_MAP } from "../ad-constants";
+import { getCreateAdFuncForType } from "./getCreateAdFuncForType";
+import { getAdId } from "./getAdId";
+
+/**
+ * getAdRegistrationID
+ *
+ * Registers instance of PBH ad function for given ad type and returns the
+ * associated ID. Used in the MakeAdComponent HOC.
+ *
+ * @param {String} adType - the type of add from the AD_TYPE enum in constants
+ * @returns {String} id
+ */
+export const getAdRegistrationID = adType => {
+  const adFunc = getCreateAdFuncForType(adType);
```

Comment:
(minor) can you rename getAdRegistrationID to make the side effect more obvious?

Reply:
Agree with thinking but deferring refactor.